xmtp_id/src/associations/serialization.rs

```diff
@@ -70,6 +70,8 @@ pub enum DeserializationError {
     InvalidAccountId,
     #[error("Invalid hash (needs to be 32 bytes)")]
     InvalidHash,
+    #[error("A required field is unspecified: {0}")]
+    Unspecified(&'static str),
```

Comment:
nitpicking: great add, wdyt of something like: `MissingField` or `RequiredFieldMissing`?

Reply:
It's originally there for the shortcomings of protos with unspecified being a required enum field, but I'm happy to generalize it if we want.